netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -340,7 +340,7 @@ private void createStream(CreateStreamCommand command, final ChannelPromise prom
       if (!connection().goAwaySent()) {
         logger.fine("Stream IDs have been exhausted for this connection. "
                 + "Initiating graceful shutdown of the connection.");
-        super.close(ctx(), ctx().newPromise());
+        close(ctx(), ctx().newPromise());
```

Comment:
oof ... was this just a bug?  Seems like we should always have called `this` version.